### PR TITLE
Initial step to adding update and insert functionality. Queries are

### PR DESCRIPTION
**Pull request by RothIOP**
currently set up to print out verbatim SQL rather than query returns for
debug purposes.

### CODE REVIEW

**Comment by RothIOP** [comment]
@.gitignore, I should have added .gitignore to my .gitignore. My bad. I just altered the example and didn't change the path and didn't want to leak credentials. 

@dockerfile We should probably split the copy step then, because when it copies in config it needs to do all installs on all builds. Makes small changes a pain in the ass. 

@Readme.md Was only trying to remove that on the fork on my side. Didn't realize that was included in the merge. 

@PostgreDBService.js I'm definitely a node novice. I just tried to follow the pattern that I saw when I was adding the POST and PUT methods. If those are handled by other modules then I can remove those. 

@ URLQueryBuilder.js That was my initial thought, but then I scrapped it because I was worried about reusing too much similar code across functions. I'll go back to that. 

This was very much WIP. I was using my fork to work as an inbetween for the VM I'm building on to test on. Didn't realize that the PR was tracking with my fork, so some of these changes were never supposed to be included.

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sudo docker build -f DockerFile -t rest-api-parser:latest .
+sudo docker stop $(sudo docker ps -a -f "ancestor=rest-api-parser:latest" -q)
+sudo docker rm $(sudo docker ps -a -f "ancestor=rest-api-parser:latest" -q)
+sudo docker run -d -p 127.0.0.1:8181:443 -p 127.0.0.1:8282:80 -p 127.0.0.1:9080:9080 rest-api-parser:latest
```
Comment (danielc2013) [inline build.sh]: Put this file in your pipeline or whatever you're using to deploy it. If I don't want to strongly tie a tagging and deployment method to the repo. There's every possibility that I'll have to change it in the future which'll break this for your stuff

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+examples/
```
Comment (danielc2013) [inline .gitignore]: I don't want to remove examples/ because others might want them locally to help them develop

File: DockerFile [hunk 3]
```diff
@@ -2,9 +2,11 @@ FROM node:8-alpine
 
 WORKDIR /usr/src/app
 
-COPY . .
-
 RUN npm install
```
Comment (danielc2013) [inline DockerFile]: What is the first npm install for? Also, you can just copy the directory which will copy package.json and then RUN npm install. That way if packages change we don't have to manage that each time in the DockerFile. If you want to leverage caching, check out this post: https://stackoverflow.com/questions/35774714/how-to-cache-the-run-npm-install-instruction-when-docker-build-a-dockerfile

**Comment by danielc2013** [comment]
If you want to do N* specific stuff that's fine. Undo the PR on this fork. Then if there's generic changes you make, do them on a separate branch (not fork) and do a PR on that branch. Then when I make updates, you can rebase (merge) my generic code into your N* code

**Comment by danielc2013** [comment]
Also, a small note... in JavaScript the `==` method of comparison is different than other languages. It won't strictly compare the values of two variables. It'll do a lazy check. If you want strict comparisons for strings you need to use `===`

**Comment by RothIOP** [comment]
Will be creating PR once work is finished on my branch.

**Pull request closed without merging**